product/swagger/products.json

```diff
@@ -0,0 +1,753 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "description": "Semana 5 - Treinamento da PagoNxt com a Alura sobre HTTP e Swagger.",
+    "version": "0.1.0",
+    "title": "Ecomm Project - Products API",
+    "termsOfService": "http://swagger.io/terms/",
+    "contact": {
+      "name": "Marina",
+      "email": "marina.mendes@pagonxt.com"
+    },
+    "license": {
+      "name": "Licença",
+      "url": "http://www.alura.com"
+    }
+  },
+  "components": {
+    "schemas": {
+      "category-response": {
+        "type": "object",
+        "properties": {
+          "_id": {
+            "type": "string"
+          },
+          "name": {
+            "type": "string"
+          },
+          "status": {
+            "type": "string",
+            "enum": [
+              "Ativa",
+              "Inativa"
+            ]
+          }
+        }
+      },
+      "category-insert": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "status": {
+            "type": "string",
+            "example": "Ativa"
+          }
+        }
+      },
+      "product-response": {
+        "type": "object",
+        "properties": {
+          "_id": {
+            "type": "string"
+          },
+          "name": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "slug": {
+            "type": "string"
+          },
+          "unit_price": {
+            "type": "number",
+            "format": "double"
+          },
+          "storage_qty": {
+            "type": "integer"
+          },
+          "category_id": {
+            "type": "string"
+          }
+        }
+      },
+      "product-insert": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "slug": {
+            "type": "string"
+          },
+          "unit_price": {
+            "type": "number",
+            "format": "double"
+          },
+          "storage_qty": {
+            "type": "integer"
+          },
+          "category_id": {
+            "type": "integer"
+          }
+        }
+      }
+    }
+  },
+  "servers": [
+    {
+      "url": null,
```

Comment:
url MUST be in the format of an URL.

Structural error at servers.0.url should be string

Reply:
url added